evap/staff/tests/test_views.py

```diff
@@ -1953,6 +1980,20 @@ def test_textanswers_full_view(self):
             page = self.app.get(self.url + '?view=full', user=self.manager, status=200)
             self.assertContains(page, self.answer)
 
+    def test_exclude_unreviewed_evaluations(self):
+        # evaluation that ended 2 days ago is definitely finished and should be proposed
+        let_user_vote_for_evaluation(self.app, self.student2, self.evaluation)
+        with run_in_staff_mode(self):
+            page = self.app.get(self.url, user=self.manager, status=200)
+            self.assertContains(page, self.evaluation2.full_name)
```

Comment:
I would shorten the comment a bit to not replicate content of the code. If someone changes the end date in `setUpTestData`, the "2" will become wrong (which I think it already is? Shouldn't it be 3?). So, I'd propose to make it something like

> evaluation2 is finished and should show up

or whatever. Also, I'd move the comment directly to the assertion doing the test

Reply:
oh you're right, I was off by one there.
it's obvoiously a good idea to avoid the calculus (at least for me) ^^